probe/main.go

```diff
@@ -66,8 +67,12 @@ func main() {
 	}
 	defer publisher.Close()
 
+	var (
+		dockerTagger *tag.DockerTagger
+		weaveTagger  *tag.WeaveTagger
```

Comment:
It is consistent with DockerTagger, which both produces a topology and tags existing topologies. I agree the naming of probe components is currently schizophrenic; that's a consequence of the organic growth of the package, which has resulted in poor separation of concerns and awkward names. I intend to address those things holistically in a refactor...